### DISCUSSION TITLE
Possible expansion of sound design?

### DISCUSSION THREAD

**Discussion by Koalemosthegod** [bangcorrupt/hacktribe#279] [Ideas]
First of all, wow, thanks for making my E2S so much better. I was wondering though, if it’s possible to access some kinda filter drive and/or the full ADSR. Having some overdrive before the filter would just be *chef’s kiss* and it seems like most Korg digital synths have it. I’m making a wild guess that the OD on the E2 Synth drums might be filter OD. 

I always thought it was really lame that the envelope costs polyphony but you don’t get to control the whole thing. 

Is it possible to add some shift+knob type functions at all? The FM, filters, fx, and grooves you added are pulling mad weight but this would be just a cherry on top.

ps: I saw where you were thinking you could get a channel to play 8 notes and that would just be disgusting. It’d turn into a really playable synth then. Micro Korg? No thanks. 8 voice Electribe 2 Sampler 😩

**Comment by bangcorrupt** [comment]
The drive on the synth PCM is a separate stage before the filter, it sounds like it uses the same algorithms as the FX sections.  I hope I can implement this some day, maybe expanded with more algorithms.  I think the envelope can be expanded to a full ADSR, maybe with options for different curve shapes, but I'm not working on this yet.

**Reply by enorrmann** [reply]
>  I think the envelope can be expanded to a full ADSR
I don't know why korg decided to omit the Sustain stage with the Amp EG active tbh
